outputs.tf

```
@@ -105,6 +105,11 @@ output "igw_arn" {
 # Publiс Subnets
 ################################################################################
 
+output "public_subnet_objects" {
+  description = "A list of all public subnets, containing the full objects."
+  value       = aws_subnet.public[*]
```

No need for the `[*]`, they can all be dropped if they are the last attribute selector
```suggestion
  value       = aws_subnet.public
```